internal/cbor/types_test.go

```diff
@@ -44,6 +44,8 @@ var booleanArrayTestCases = []struct {
 	binary string
 	json   string
 }{
+	{[]bool{}, "\x9f\xff", "[]"},
```

Comment:
Noticed when I was doing _my_ tests, that we didn't have a test for and empty or single-entry array... empty arrays are special-cased in cbor, so added that test. Single entry arrays don't have special-case (like json does), but added a case anyway.